configurations/performance/cassandra_stress_gradual_load_steps.yaml

```diff
@@ -1,4 +1,8 @@
 # Define load ops for steps
-perf_gradual_threads: {"read": 620, "write": 400, "mixed": 1900}
+# # For debugging, you can set a specific thread count for each step (per load).
+## The value of perf_gradual_threads[load] must be either:
+##   - a single-element list (applied to all throttle steps), or
+##   - a list with the same length as perf_gradual_throttle_steps[load] (one thread count per step).
```

Comment:
and what happens if it's not one element and doesn't match the size of  `perf_gradual_throttle_steps` ?

Reply:
good question. I added validation in `sct_config.py`